clevis/Sanity/password-generation/main.fmf

```diff
@@ -0,0 +1,18 @@
+summary: Generating password during clevis binding does not fail on low entropy
+test: ./test.sh
+recommend:
+  - clevis
+  - clevis-luks
+  - tang
+link:
+  - verifies: https://bugzilla.redhat.com/show_bug.cgi?id=2207488
+adjust:
+  - enabled: false
```

Comment:
Is not distro < rhel-9 redundant? I mean, distro < rhel-9.3 does not include also distro < rhel-9

Reply:
I checked the behavior and `distro < rhel-9.3` indeed should cover it. I put this from historical reason when relevancy "engine" behaved differently.

Reply:
OK. Thanks